radio/src/opentx.cpp

```diff
@@ -365,7 +365,7 @@ void generalDefault()
   g_eeGeneral.pwrOffSpeed = 2;
 #endif
 
-#if defined(MANUFACTURER_RADIOMASTER)
+#if defined(MANUFACTURER_RADIOMASTER) || defined(MANUFACTURER_JUMPER)
```

Comment:
Why not simply `AUDIO_MUTE_GPIO`? It seems most people want the mute enabled to stop noise when not speaking/playing sound. 
```suggestion
#if defined(AUDIO_MUTE_GPIO)
```

Reply:
Not too sure, that would affect X10 which does have this pin, but seems to be ok with mute deactivated (which you really want UNLESS..). We could consider using default module type instead, unsure